imgui/core.pyx

```diff
@@ -2201,6 +2205,86 @@ cdef class _ImGuiViewport(object):
         self._require_pointer()
         return _cast_ImVec2_tuple(self._ptr.GetWorkCenter())
 
+
+cdef class ImGuiWindowClass(object):
+    """
+    // [ALPHA] Rarely used / very advanced uses only. Use with SetNextWindowClass() and DockSpace() functions.
+    // Important: the content of this class is still highly WIP and likely to change and be refactored
+    // before we stabilize Docking features. Please be mindful if using this.
+    // Provide hints:
+    // - To the platform backend via altered viewport flags (enable/disable OS decoration, OS task bar icons, etc.)
+    // - To the platform backend for OS level parent/child relationships of viewport.
+    // - To the docking system for various options and filtering.
```

Comment:
Considering the docking branch is WIP, and this api is alpha, it is very likely to change in the future.
In a case like that, it would make sense (imo) to not include it?

Reply:
Yes, I do agree. However many features introduced in the docking branch are in beta and may also change. I am not against mapping alpha/beta functionalities as long as it is properly documented. We can always change and document our interface in a future version as well.

Reply:
I agree as well and going to comment it, maybe it will be useful in the next future. BTW, I have added it since it is used by `SetNextWindowClass(const ImGuiWindowClass* window_class)` explicitly.